src/main/java/org/spongepowered/mod/mixin/core/event/player/MixinEventPlayerInteractBlock.java

```diff
@@ -79,4 +85,15 @@ public EntityInteractionType getInteractionType() {
         return Optional.fromNullable(new Cause(null, this.entityPlayer, null));
     }
 
+    @Override
+    public Optional<Vector3d> getClickedPosition() {
+        return Optional.of(VecHelper.toVector(this.pos).toDouble());
```

Comment:
The problem is that the event itself gets called from ```ForgeEventFactory```. If I modify the event object in ```MixinNetHandlerPlayServer```. The event will be called already. So that means I need to make a mixin for ```ForgeEventFactory```. But even that wouldn't work, as I haven't found anything in mixins to change/modify code.